src/nunit-gui/Views/SettingsPages/GuiSettingsPage.cs

```diff
@@ -28,11 +28,11 @@
 using System.Data;
 using System.Text;
 using System.Windows.Forms;
+using NUnit.Gui.Model;
```

Comment:
In the coding standards you mentioned putting NUnit usings inside the namespace. Are we moving away from that? (I ask that because you moved the Model.Settings up here.